src/libraries/System.Net.Quic/src/System/Net/Quic/QuicStream.cs

```diff
@@ -543,12 +543,12 @@ private unsafe int HandleEventShutdownComplete(ref SHUTDOWN_COMPLETE data)
                 (shutdownByApp: true, closedRemotely: true) => ThrowHelper.GetConnectionAbortedException((long)data.ConnectionErrorCode),
                 // It's local shutdown by app, this side called QuicConnection.CloseAsync, throw QuicError.OperationAborted.
                 (shutdownByApp: true, closedRemotely: false) => ThrowHelper.GetOperationAbortedException(),
-                // It's remote shutdown by transport, (TODO: we should propagate transport error code), throw QuicError.InternalError.
+                // TODO: we should propagate transport error code
```

Comment:
Is the removal of the description `// It's remote shutdown by transport,` in the comment intentional? I don't mind it since the named variables are pretty self-explanatory. But if we're doing it, could you do it for all the cases?